consensus/enclave/mock/src/mock_consensus_enclave.rs

```diff
@@ -23,7 +23,7 @@ use mockall::*;
 
 mock! {
     pub ConsensusEnclave {} // This is used to generate a MockConsensusEnclave struct.
-    trait ConsensusEnclave {
+    impl ConsensusEnclave for ConsensusEnclave {
```

Comment:
`impl X for X` feels weird, but if it works I'm not gonna complain.

Reply:
Yeah, totally with you on that one. Yhey're silently adding a "Mock" prefix to everything, so if you want a `MockConsensusEnclave` to implement the `ConsensusEnclave` trait, this is how you have to do it.